docs/validation/daily-report/2023-11-27/index.md

```diff
@@ -0,0 +1,92 @@
+# 2023-11-27: Transect NSW Site 1, Sentinel 2b overpass
```

Comment:
Should "Sentinel 2b" be capitalised or hyphenated? See these examples:

https://pr-179-preview.khpreview.dea.ga.gov.au/data/category/sentinel-2b-analysis-ready-data/
https://pr-179-preview.khpreview.dea.ga.gov.au/data/product/dea-surface-reflectance-sentinel-2b-msi/?tab=overview